makeDebuggable.py

```diff
@@ -558,10 +558,42 @@ def patchApk(fnIn, fnOut, keystore, keyAlias):
     inZip.close()
     outZip.close()
 
-    print("Signing ...")
+    outZipAligned = outZip.filename[:-4] + "-aligned.apk"
```

Comment:
It's probably better to use an intermediate temporary file for `outZip.filename` and use the name passed on the command line as the final aligned and signed output.